src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,8 +2,11 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  if (src.includes('urn:') || src.includes('localhost:')) return src;
-  const urlParts = src.replace(/https?:\/\//g, '').split('/');
+  const imageSrc = !src
+    ? 'https://ichef.bbci.co.uk/images/ic/640xn/p0b36kgx.png'
+    : src;
```

Comment:
I find "positive" ternary statements easier to read/understand than "negative" ones. But also we could simplify this into 1 line:

```suggestion
  const imageSrc = src || 'https://ichef.bbci.co.uk/images/ic/640xn/p0b36kgx.png';
```

or (I never know the difference / which is best...? Maybe this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)

```suggestion
  const imageSrc = src ?? 'https://ichef.bbci.co.uk/images/ic/640xn/p0b36kgx.png';
```